tpm_futurepcr/LogEvent.py

```diff
@@ -1,12 +1,17 @@
+import hashlib
+from abc import abstractmethod, ABC
 from dataclasses import dataclass, InitVar, field
 import dataclasses
+from pathlib import Path
 from pprint import pformat
 from typing import Any
 
+import signify
```

Comment:
We need `signify.fingerprinter` specifically, as the parent signify module doesn't implicitly import sub-modules.

```
py3.10 >>> import signify
py3.10 >>> signify.fingerprinter.AuthenticodeFingerprinter
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: module 'signify' has no attribute 'fingerprinter'
```

hashlib and signify imports should be sorted together with the rest.